UnitsNet.Tests/UnitLocalizedNamesCacheFixture.cs

```diff
@@ -0,0 +1,15 @@
+﻿using Xunit;
+
+namespace UnitsNet.Tests
+{
+    [CollectionDefinition(nameof(UnitLocalizedNamesCacheFixture), DisableParallelization = true)]
```

Comment:
CollectionDefininition itself is used to shared a context (data for example between a collection) of tests . The parallelization is just an additionnal option. 
I think that:
1°) you should add a getter in UnitLocalizedNamesCacheFixture to get the Culture instance
2°) you should modify the constructors of all your TUs to inject the fixtue each time you need to get the culture

Links:
[https://xunit.net/docs/shared-context](https://xunit.net/docs/shared-context)
[https://blog.somewhatabstract.com/tag/collectiondefinition/](https://blog.somewhatabstract.com/tag/collectiondefinition/)

Reply:
Interesting, could you give an example of how this could be used to test `ToString()` and `Parse()`, where no culture is given as parameter, but we want a consistent way to test specific cultures in the current thread? I'm very interested to learn if there is a better way to do this.